### ISSUE TITLE
Add multiple tags with a single function call in Pyaqueduct

### ISSUE THREAD

**Issue by samiralavi** [AqueductHub/pyaqueduct#12]
as a user, I want to be able to add a list of tags to the experiments with one function call.

which might be good to add also add_tags function which takes a list of tags. and it is worth deprecating add_tag in the future major release with a warning being said previously.

**Comment by samiralavi** [comment]
@str-anger, to implement this functionality, Aqueduct Core should support adding multiple tags to the experiment in one mutation call. At this time the graphQL API only supports adding one tag at a time. It is possible to write a loop in PyAqueduct to call that multiple times, but this makes the transaction not atomic. If during adding multiple tags, something fails, the call becomes some of the tags will be there and some not, which is against the transactional behaviour of PyAqueduct APIs.

I am moving this tasks to blocked until and create a new issue on Aqueduct Core to repo.

**Comment by samiralavi** [comment]
Blocked by : https://github.com/AqueductHub/aqueductcore/issues/74
